cli/app/commands/conflict/models.py

```diff
@@ -0,0 +1,24 @@
+"""
+Data models and configuration for the conflict command.
+"""
+
+from typing import Optional
+from pydantic import BaseModel, Field
+
+
+class ConflictCheckResult(BaseModel):
```

Comment:
we don't have models.py in any other commands, so might be better to remove it and move logics out of this specific file

Reply:
Keep it as part of the existing logic in conflict.py?